src/background.ts

```diff
@@ -17,10 +17,22 @@ const theNaughtyList: Record<string, string[]> = {
   TruAbilities: ["*://*.truabilities.com/*"],
   MaxAccess: ["*://*.maxaccess.io/*"],
   User1st: ["*://*.user1st.info/*"],
+  DIGIaccess: ["*://*.digiaccess.org/*"],
```

Comment:
```suggestion
  Accessibly: ["*://*.accessiblyapp.com/*"],
  "Eye-Able": ["*://cdn.eye-able.com/*"],
  DIGIaccess: ["*://*.digiaccess.org/*"],
```

There have been further changes in `master`.